### PR TITLE
Orbit.sample can now recieve a different propagator

### PR DESCRIPTION
**Pull request by shreyasbapat**
Issue #286 Addressed

### CODE REVIEW

**Comment by shreyasbapat** [comment]
I will resolve these failing CI tests tonight!

**Comment by Juanlu001** [comment]
Before you go on, I think this needs a rebase again :)

File: src/poliastro/twobody/orbit.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 from poliastro.constants import J2000
 from poliastro.twobody.angles import nu_to_M
-from poliastro.twobody.propagation import propagate
+from poliastro.twobody.propagation import propagate, cowell
```
Comment (Juanlu001) [inline src/poliastro/twobody/orbit.py]: Where are you using the `cowell` function? And on the other hand, do you plan to add it for testing?

**Comment by Juanlu001** [comment]
Never mind, I'm merging this :) Thanks!

File: src/poliastro/twobody/orbit.py (hunk 1 above)
Comment (shreyasbapat) [reply src/poliastro/twobody/orbit.py]: Yeah! I plan to do so. Currently, not using it

**Pull request merged**